contracts/l2/governance/l2_governance_relay.cairo

```diff
@@ -43,8 +38,15 @@ func relay{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
 ) {
```

Comment:
we could rename `spell` to `spell_hash`

Reply:
I want it to be consistent with the l1 payload.